### PR TITLE
updated readme with how to select model

### LINKED ISSUE #1: A way to select which model?
I tried looking around in the code and think that running `openai-retrieve-model` let's you select which model to use? I'm not totally sure though. If that is correct I can open a PR with that info in the README.

### PR DESCRIPTION
**Pull request by chookity-pokk**
Tried to add a bit of information on how to choose your model as well as the necessary code. Let me know if anything is missing or needs to get fixed!

Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ applications written in the Elips language.
...
 
+You can also choose which model you want to use by going to the (api)[https://api.openai.com/v1/models] website and looking at the id's. For 
+code usage you probably want something that starts with `code-` whereas with more text related files you'll likely want something starting with
+`text-`. 
+
+```elisp
+(setq openai-completion-model "NAME-HERE")
+```
+
+
```
Comment (jcs090218) [inline README.md:36]: Some nits: 

1. Let's not have extra newline here. 
2. Keep the column within `80`.

**Review by jcs090218 — COMMENTED** [review]
Some nits.

Thanks for the changes! ❤️ This package is still WIP, feel free to open issues or PRs! ;)

File: README.md (hunk 1 above)
Comment (chookity-pokk) [reply README.md]: I believe the commit I just pushed fixes these errors.

**Comment by jcs090218** [comment]
Thank you! ❤️ 😄

**Pull request merged**